src/libraries/Microsoft.Extensions.Configuration.Binder/src/Microsoft.Extensions.Configuration.Binder.csproj

```diff
@@ -37,6 +37,12 @@
 
   <PropertyGroup>
     <IncludeMultiTargetRoslynComponentTargets>false</IncludeMultiTargetRoslynComponentTargets>
-    <RoslynComponentsAreOffByDefault>true</RoslynComponentsAreOffByDefault>
+    <EnableConfigurationBindingGeneratorTargetsFilePath>buildTransitive\EnableConfigurationBindingGenerator.targets</EnableConfigurationBindingGeneratorTargetsFilePath>
   </PropertyGroup>
+
+  <ItemGroup>
+    <Content Include="$(EnableConfigurationBindingGeneratorTargetsFilePath)" PackagePath="buildTransitive\netstandard2.0\$(PackageId).targets" />
```

Comment:
Can this look like the docs and the existing libraries?

https://github.com/dotnet/runtime/blob/4f4dd93eca92aca3c6bd49a5b9c2eca908112c32/src/libraries/Microsoft.Extensions.Configuration.UserSecrets/src/Microsoft.Extensions.Configuration.UserSecrets.csproj#L19-L24

https://github.com/dotnet/runtime/blob/4f4dd93eca92aca3c6bd49a5b9c2eca908112c32/docs/coding-guidelines/libraries-packaging.md?plain=1#L130-L137